docs/source/distributed.pipelining.rst

```diff
@@ -176,3 +176,16 @@ Note that since we split our model into three stages, we must run this script wi
 .. code-block:: bash
 
   torchrun --nproc_per_node=3 example.py
+
+Microbatch Utilities
+====================
+
+.. automodule:: torch.distributed.pipelining.microbatch
+
+.. currentmodule:: torch.distributed.pipelining.microbatch
+
+.. autoclass:: TensorChunkSpec
```

Comment:
nit: May want to have block comment in class definition describing the class so it shows up in the docs.